benchmarks/cp/test_gdn_with_cp.py

```
@@ -0,0 +1,924 @@
+
+sys.path.append("../../")
+
+
+try:
+    import fused_weight_gradient_mlp_cuda
+    apex_func = None
+    if hasattr(fused_weight_gradient_mlp_cuda, "wgrad_gemm_accum_fp32"):
+        apex_func = fused_weight_gradient_mlp_cuda.wgrad_gemm_accum_fp32
+    if hasattr(fused_weight_gradient_mlp_cuda, "fused_wgrad_gemm_accum_fp32"):
+        apex_func = fused_weight_gradient_mlp_cuda.fused_wgrad_gemm_accum_fp32
+    from functools import partial
```

Flake8 (7.3.0)

[error] redefinition of unused 'partial' from line 24

(F811)